### ISSUE TITLE
gcc48 isn't available on brew anymore

### ISSUE THREAD

**Issue by Bimde** [OpenShot/libopenshot#203]
`brew install gcc48` doesn't work and the `homebrew/versions` tap which had it doesn't exist anymore. Am I doing something wrong?

```
Error: No available formula with the name "homebrew/core/gcc@4.8" 
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
```

Mac version: 10.14.3

**Comment by Bimde** [comment]
On another note, I'm trying to use the library for a personal project. If I find something I'd like to improve I may come back to contribute later, but for now, is using [this](https://github.com/OpenShot/libopenshot/wiki/Mac-Build-Instructions) installation guide the best way to set it up for non-contributors?

**Comment by ferdnyc** [comment]
@Bimde 

That wiki page is going to be your best bet, yeah. With the one caveat that, if you're planning to make changes to the libopenshot code itself, you're better off forking this repo, then cloning and building your _fork_ instead. That way, any changes just need to be pushed to your fork to make them PR-ready.

**Comment by ferdnyc** [comment]
As far as `gcc48` goes, I'm not a Mac user but on Fedora we build with whatever recent GCC is the distro default, which for Fedora 29 is currently GCC 8.3.1. If there's a gcc newer than 4.8 available, I can't imagine why that _wouldn't_ work as a replacement.

**Comment by ferdnyc** [comment]
Closing as hopefully answered. I've edited the wiki page in question to remove references to gcc48 (hopefully a more current version will be usable for building libopenshot, I can't imagine why it wouldn't), and to note that the exact path names there reference specific versions of tools and are surely now out of date.

I think hoping to keep a document like that current when it would require updating every time a new version of any dependency is released... well, it's an ambitious idea, but unrealistic. Perhaps some more generic approach can be found, suggestions or edits welcome.

Closing this as answered best we can.
